source/time_tables/departure_boards.cpp

```diff
@@ -78,8 +78,14 @@ render_v1(const std::map<uint32_t, pbnavitia::ResponseStatus>& response_status,
         //Now we fill the date_times
         for(auto dt_st : id_vec.second) {
             auto date_time = schedule->add_date_times();
+            auto vj = dt_st.second->vehicle_journey;
```

Comment:
I think we should test that stop_time is not null (as it is done in fill_pb_object).